### DISCUSSION TITLE
Filters webpart customization for complex filters display

### DISCUSSION THREAD

**Discussion by arunakshi** [microsoft-search/pnp-modern-search#3693] [General]
Looking for filters webpart customization where I want to display 3 filters display next to each other. Like I want to display Taxonomy filters in dropdown. For customizing via custom handlebar templates works for simple filters but I have above requirements  with complex filters. Can we create custom filter cards? 

![image](https://github.com/microsoft-search/pnp-modern-search/assets/12812202/869efa1c-c1b3-453c-a21c-c5a9f937b994)

**Comment by wobba** [comment]
Have you tried the horizontal layout?
<img width="336" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/51104/dca1990f-9e03-426c-872b-2c3db7d124eb">
